src/app/ChatView.tsx

```
@@ -47,7 +76,7 @@ export const ChatView = ({
   );
 };
 
-const ChatMessage = ({ role, content }: { role: string; content: string }) => {
+const ChatMessage = ({ role, content, loading }: { role: string; content: string, loading: boolean }) => {
```

ESLint

[error] 'loading' is defined but never used.

(@typescript-eslint/no-unused-vars)